ui/de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -353,8 +351,13 @@ public void onShareSelected(List<Analysis> selected) {
 
     @Override
     public void setCurrentFilter(AnalysisFilter filter) {
-       this.currentFilter = filter;
-       loadAnalyses(currentFilter);
+        if(filter == null) {
+            currentFilter =filter;
+            return;
```

Comment:
Not knowing the code that well so bear with me, but why do we not call `loadAnalyses` in the case of setting the filter to null (which I assume is intended to un-set the filter)?